Source/Core/Core/PowerPC/JitArm64/JitArm64_Integer.cpp

```diff
@@ -368,7 +368,17 @@ void JitArm64::cntlzwx(UGeckoInstruction inst)
 
   if (gpr.IsImm(s))
   {
-    gpr.SetImmediate(a, __builtin_clz(gpr.GetImm(s)));
+    const u32 imm = gpr.GetImm(s);
+#ifdef _MSC_VER
```

Comment:
Please just avoid the intrinsic here. This can be done in pure C++

Reply:
what will generate `clz`?

Reply:
I found `_CountLeadingZeros()` in `intrin.h`. But yeah, could be worth adding a helper in `BitUtils` in the future.